sdk/documenttranslation/azure-ai-documenttranslation/azure/ai/documenttranslation/_models.py

```diff
@@ -57,15 +56,16 @@ def _to_generated_list(glossaries):
 class TranslationTarget(object):  # pylint: disable=useless-object-inheritance
     """Destination for the finished translated documents.
 
-    :param target_url: Required. Location of the folder / container with your documents.
-    :type target_url: str
-    :param language_code: Required. Target Language Code.
-    :type language_code: str
-    :keyword str category_id: Category / custom system for translation request.
-    :keyword glossaries: List of TranslationGlossary.
+    :param str target_url: Required. The target location for your translated documents.
+        This should be a container SAS URL to your target container.
+    :param str language_code: Required. Target Language Code. This is the language
+        you want your documents to be translated to. See supported languages here:
+        https://docs.microsoft.com/azure/cognitive-services/translator/language-support#translate
+    :keyword str category_id: Category / custom model ID for using custom translation.
```

Comment:
Consider adding `Optional`

Reply:
ohhh or not needed because it is part of `kwargs` ? I guess I got confused because `glossaries` says `Optional glossaries....`
whatever you think is better for Python

Reply:
Optional is kind of redundant here so I'm just going to remove it from glossaries. Thanks for catching that!